npm ERR! Path must be a string. Received undefined

This is likely due to serializable depending on an old version of get-parameter-names, which in turn depends on testla which fails to install. [npm-debug.log](https://gist.github.com/04592c65bff6c7dbb636)

```
$ npm install serializable
npm ERR! Darwin 14.5.0
npm ERR! argv "/usr/local/bin/iojs" "/usr/local/bin/npm" "install" "serializable"
npm ERR! node v4.1.1
npm ERR! npm  v3.3.5

npm ERR! Path must be a string. Received undefined
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/dan/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -23,7 +23,7 @@
   "dependencies": {
     "mixto": "1.x",
     "underscore-plus": "1.x",
-    "get-parameter-names": "~0.1.2"
+    "get-parameter-names": "~0.2.0"
   },
   "devDependencies": {
     "coffee-script": "~1.6.3",

